src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,146 @@
+package duke.util;
+
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Represents the storage which stores the lists inside the disk.
+ */
+public class Storage {
+    private ArrayList<String> listStates= new ArrayList<>();
+    private static String listFilePath = "./data/duke.txt";
+
+    /**
+     * Access the stored lists and parse it into usable ArrayList of Task.
+     * Each line of the stored file is read and parsed using a Parser.
+     * The listStates is changed with each task read to capture the file state.
+     *
+     * @return the converted Task ArrayList from the stored data.
+     */
+    public ArrayList<Task> loadTasks() {
+        ArrayList<Task> temp = new ArrayList<>();
+        this.initializeListFile();
+        File f = new File(listFilePath);
+        try{
+            Scanner s = new Scanner(f);
+            while (s.hasNext()) {
+                String str = s.nextLine();
+                listStates.add(str);
+                Parser.initializeTask(str,temp);
+            }
+        }catch (FileNotFoundException e){
+            System.out.println("FILE NOT FOUND");
+        }
```

Comment:
This seems to violate the coding styles for not adding whitespaces?

```suggestion
        try {
            Scanner s = new Scanner(f);
            while (s.hasNext()) {
                String str = s.nextLine();
                listStates.add(str);
                Parser.initializeTask(str,temp);
            }
        } catch (FileNotFoundException e){
            System.out.println("FILE NOT FOUND");
        }
```